.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
     - stable
-
+script:
+  - node ./validation/markets-validator.js
```

Comment:
Since you have updated `package.json` so that `npm test` also runs the validation I think you can undo your changes to `.travis.yml`.